.github/workflows/security.yaml

```diff
@@ -4,8 +4,8 @@ on:
   push:
     branches: [main]
   pull_request:
-    # The branches below must be a subset of the branches above
-    branches: [main]
+    paths:
+      - "package.json"
```

Comment:
Should we add to main as well?  Not a huge deal but it does kinda give a bad look to have seemingly failing CI on main.

Reply:
Oh yeah good call!